modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/PDFProcessorImpl.java

```diff
@@ -120,6 +128,32 @@ public void generateImages(
 			sourceFileVersion, destinationFileVersion, maxNumberOfPages);
 	}
 
+	@Override
+	public void generatePDFPreviews(long userId) {
```

Comment:
Why do we need the user ID here? Can we use the system (default) user?

Reply:
Ok, I'll use UserConstants.USER_ID_DEFAULT